File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    s = s.toLowerCase()
    let p = 0
    let y = 0
    for (i = 0; i <= s.length; i++)
        if (s[i] === 'p') {
            p += 1
        } else if (s[i] === 'y') {
            y += 1
        }
    
    if (p === y) {
        return true
    } else {
        return false
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        s = s.toUpperCase();
        int cnt1 = 0;
        int cnt2 = 0;
        for (char x : s.toCharArray()) {
            if (x == 'P') cnt1++;
            else if (x == 'Y') cnt2++;
        }
        return cnt1 == cnt2;
    }
}
